--- /usr/local/share/X11/xkb/symbols/pc.orig	2024-09-09 23:06:49.185779000 +0300
+++ /usr/local/share/X11/xkb/symbols/pc	2024-09-09 23:35:56.994898000 +0300
@@ -31,6 +31,8 @@
     modifier_map Control { Control_L, Control_R };
     modifier_map Mod1    { Alt_L, Alt_R, Meta_L, Meta_R };
     modifier_map Mod2    { Num_Lock };
+    // Right Control as i3wm super key
+    modifier_map Mod3    { Control_R };
     modifier_map Mod4    { Super_L, Super_R };
 
     // Six fake keys for virtual<->real modifiers mapping:
